src/modules/matches/dto/match.dto.ts

```diff
@@ -0,0 +1,36 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { Match } from '../match.entity';
+import { TeamDto } from '../../teams/dto';
+
+export class MatchDto {
+  @ApiProperty({ type: String })
+  id: string;
+
+  @ApiProperty({ type: TeamDto })
+  home_team: TeamDto;
+
+  @ApiProperty({ type: Number })
+  home_score: number;
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏Do you think it is relevant to add a minimun properties into the ApiProperty decorator ?